site/content/docs/v1.6/restic.md

```diff
@@ -492,7 +492,7 @@ on that node. The controller executes `restic restore` commands to restore pod v
     check it for integrity)
 1. Velero adds an init container to the pod, whose job is to wait for all restic restores for the pod to complete (more
 on this shortly)
-1. Velero creates the pod, with the added init container, by submitting it to the Kubernetes API
+1. Velero creates the pod, with the added init container, by submitting it to the Kubernetes API. Then, Kubernetes scheduler schedule this pod to a worker node, and the pod must be in a running state. If the pod fails to start for some reason (i.e. lack of cluster resources), the Restic restore will not be done."
```

Comment:
"Then, Kubernetes scheduler schedule this pod to a " -> "Then, the Kubernetes scheduler schedules this pod to a"